src/bin/ion/commands/beta/inspect.rs

```diff
@@ -19,31 +19,31 @@ const ABOUT: &str =
 // Creates a `clap` (Command Line Arguments Parser) configuration for the `inspect` command.
 // This function is invoked by the `inspect` command's parent, `beta`, so it can describe its
 // child commands.
-pub fn app() -> App<'static, 'static> {
-    App::new("inspect")
+pub fn app() -> Command {
+    Command::new("inspect")
         .about(ABOUT)
         .arg(
-            Arg::with_name("output")
+            Arg::new("output")
                 .long("output")
-                .short("o")
-                .takes_value(true)
+                .short('o')
```

Comment:
🗺️ The `short` name method takes a `char` instead of a `&str`. I didn't investigate the motivation.